test/views/declaration/RepresentativeDetailsViewSpec.scala

```diff
@@ -32,86 +31,34 @@ import views.tags.ViewTest
 @ViewTest
 class RepresentativeDetailsViewSpec extends UnitViewSpec with ExportsTestData with Stubs with Injector {
 
-  private val page = new representative_details(mainTemplate)
+  private val page = instanceOf[representative_details]
   private val form: Form[RepresentativeDetails] = RepresentativeDetails.form()
   private def createView(mode: Mode = Mode.Normal, form: Form[RepresentativeDetails] = form): Document =
     page(mode, form)(journeyRequest(), stubMessages())
 
   "Representative Details View on empty page" should {
     val view = createView()
 
-    "have proper messages for labels" in {
```

Comment:
Please stay with this test, but test only messages keys that you're using on the page